server/compute.js

```diff
@@ -20,6 +22,46 @@ function applyUtils(sandbox = {}) {
   sandbox._ = deepFreeze(lodash);
 }
 
+const buildPayload = (pld, pl = {}) => {
```

Comment:
Yes, you are right, mostly the preview forces us to do all that data parsing here. Sorry about that.